### DISCUSSION TITLE
✅ 🔗 Adding a placeholder for margin and padding in tablet and mobile editing (Duplicate of #3356)

### DISCUSSION THREAD

**Discussion by incircolo** [elementor/elementor#14162] [Feature Request]
I think it would be very useful to be able to see values in padding and margin fields inherited from the desktop version (as a placeholder?), as sometime it'is not so easy to make corrections in tablet or mobile version.

PS.
This would be useful also for "typography" text section and more, i think.

**Comment by nicholaszein** [comment]
This feature was added to Elementor in version 3.4.0.
